packages/pipeline/src/scripts/pull_github.ts

```diff
@@ -0,0 +1,77 @@
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+import { logUtils } from '@0x/utils';
+
+import { GithubSource } from '../data_sources/github';
+import { GithubFork, GithubPullRequest, GithubRepo } from '../entities';
+import * as ormConfig from '../ormconfig';
+import {
+    enrichGithubForkWithComparisonDetails,
+    parseGithubForks,
+    parseGithubPulls,
+    parseGithubRepo,
+} from '../parsers/github';
+import { handleError } from '../utils';
+
+const GITHUB_OWNER = '0xProject';
+const GITHUB_REPO = '0x-monorepo';
+const GITHUB_BRANCH = 'development';
+const RECORDS_PER_PAGE = 100;
+
+let connection: Connection;
+
+(async () => {
+    connection = await createConnection(ormConfig as ConnectionOptions);
+    const accessToken = process.env.GITHUB_ACCESS_TOKEN;
+    if (accessToken === undefined) {
+        throw new Error('Missing required env var: BLOXY_API_KEY');
```

Comment:
Change `BLOXY_API_KEY` to `GITHUB_ACCESS_TOKEN`

Reply:
Ha, thanks!